ci_config.json

```diff
@@ -47,6 +48,12 @@
       "libvulkan-dev"
     ]
   },
+  "jfinkhaeuser-json-rpc": {
+    "build_options": [
+      "cpp_std=c++17"
+    ],
+    "fatal_warnings": false
+  },
```

Comment:
What is the purpose of disabling warnings here?

Reply:
I use specific warning flags rather than relying on meson doing what I want it to do.

Reply:
Do you have a specific concern about what meson may do here?